packages/cli/src/templates/dev.entry.js

```diff
@@ -22,5 +21,16 @@ registerHook(AvailableHooks.ON_GENERATION, ({ input, output, metric, runId, agen
 });
 
 registerHook(AvailableHooks.ON_EVALUATION, async traceObject => {
-  evalStore.push(traceObject);
+  if (mastra && mastra.memory && mastra.memory.storage) {
```

Comment:
```suggestion
  if (mastra?.memory?.storage) {
```

Reply:
Oh yeah I wrote it like that because the optional chains were disappearing in the .mastra folder. But i'll try it again and see what exactly is going on

Reply:
Nevermind this isn't happening anymore. I'll switch to optional chaining.